Gopkg.toml

```diff
@@ -30,14 +30,6 @@ ignored = ["github.com/kata-containers/tests/kata-webhook"]
   name = "github.com/urfave/cli"
   version = "1.20.0"
 
-[[constraint]]
```

Comment:
Hi @snir911 , asking just for curiosity: those constraints are used on log-parser so they are no longer used or they are leftovers of another clean ups?

Reply:
leftovers